src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -175,35 +191,294 @@ type SingleBlastJobResultProps = {
+  {
+    columnId: 'id',
+    title: 'ID',
+    isFilterable: false,
+    isHideable: false
+  },
+  {
+    width: '130px',
+    columnId: 'e_value',
+    title: 'E-value',
+    isSortable: true,
+    helpText: (
+      <span>
+        The number of times a match is expected to occur by chance <br></br>
+        Typically, a low e-value indicates greater similarity between sequences
+      </span>
+    )
+  },
+  {
+    width: '100px',
+    columnId: 'hit_length',
+    title: 'Length',
+    isSortable: true
+  },
+  { width: '200px', columnId: 'view_alignment', isHideable: false },
+  {
+    width: '100px',
+    columnId: 'percentage_id',
+    title: '% ID',
+    isSortable: true,
+    helpText: (
+      <span>
+        Indicates the extent to which the query sequence and the hit have the
+        same residues at the same positions in an alignment
+      </span>
+    )
+  },
+  {
+    width: '100px',
+    columnId: 'score',
+    title: 'Score',
+    isSortable: true
+  },
+  {
+    width: '300px',
+    columnId: 'genomic_location',
+    title: 'Genomic location',
+    helpText: <span>Location of the hit in this species</span>
+  },
+  {
+    width: '100px',
+    columnId: 'hit_orientation',
+    title: 'Hit orientation',
+    isSortable: true,
+    helpText: (
+      <span>The orientation of the hit against the query sequence</span>
+    ),
+    bodyCellClassName: styles.strandColumn
+  },
+  {
+    width: '130px',
+    columnId: 'hit_start',
+    title: 'Hit start',
+    isSortable: true,
+    helpText: (
+      <span>
+        The position within the target sequence at which the hit started
+      </span>
+    )
+  },
+  {
+    width: '130px',
+    columnId: 'hit_end',
+    title: 'Hit end',
+    isSortable: true,
+    helpText: (
+      <span>
+        The position within the target sequence at which the hit ended
+      </span>
+    )
+  },
+  {
+    width: '130px',
+    columnId: 'query_start',
+    title: 'Query start',
+    isSortable: true,
+    helpText: (
+      <span>
+        The first position within the query sequence that matches the beginning
+        of the hit that BLAST returns
+      </span>
+    )
+  },
+  {
+    width: '130px',
+    columnId: 'query_end',
+    title: 'Query end',
+    isSortable: true,
+    helpText: (
+      <span>
+        The last position within the query sequence that matches the end of the
+        hit that BLAST returns
+      </span>
+    )
+  }
+];
+
 const SingleBlastJobResult = (props: SingleBlastJobResultProps) => {
-  const { species: speciesInfo, diagramWidth } = props;
+  const { species: speciesInfo, diagramWidth, blastDatabase } = props;
   const { data } = useFetchBlastSubmissionQuery(props.jobId);
 
+  const [isExpanded, setExpanded] = useState(false);
+
+  const [tableState, setTableState] = useState<Partial<DataTableState>>({
+    rowsPerPage: 100,
+    sortedColumn: {
+      columnId: 'e_value',
+      sortedDirection: SortingDirection.DESC
+    }
+  });
```

Comment:
https://user-images.githubusercontent.com/6834224/188636052-ec78afb5-9001-4a78-8601-aed1930135c1.mov

The default sort by e-value should the the opposite — from smallest to largest

Reply:
That's done